libs/dh/shared/assets/src/assets/configuration/dh-b2c-environment.local.json

```diff
@@ -1,6 +1,6 @@
 {
-  "clientId": "88e5d356-0c71-49e9-b260-d0629f3c0445",
-  "backendId": null,
-  "authority": "https://dev002DataHubB2C.b2clogin.com/dev002DataHubB2C.onmicrosoft.com/B2C_1_sign_in_experiments",
-  "knownAuthorities": ["dev002DataHubB2C.b2clogin.com"]
+  "clientId": "d91c10bb-1441-4ae5-9bf9-e6845567d018",
+  "backendId": "d91c10bb-1441-4ae5-9bf9-e6845567d018",
+  "authority": "https://devdatahubb2c.b2clogin.com/4a7411ea-ac71-4b63-9647-b8bd4c5a20e0/B2C_1_u001_signin",
+  "knownAuthorities": ["devdatahubb2c.b2clogin.com"]
```

Comment:
question: is it safe to commit this?

Reply:
Nope, should not have been committed, good catch